src/didstore.c

```diff
@@ -1569,13 +1627,11 @@ bool DIDStore_ContainsPrivateKey(DIDStore *store, DID *did, DIDURL *id)
     return true;
 }
 
-int DIDStore_StorePrivateKey(DIDStore *store, const char *storepass, DID *did,
-        DIDURL *id, const uint8_t *privatekey)
+int DIDStore_StorePrvKey(DIDStore *store, DID *did, DIDURL *id, const char *prvkey)
```

Comment:
Will export DIDStore_StorePrvKey to ela_did.h as exported API? if not , better to use 
"DIDStore_StorePrivateKey_Internal" according to your name convention.